.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+    branches:
+      - stage
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pull-request
+        uses: actions/checkout@v3
+      - name: Create Pull Request
+        uses: diillson/auto-pull-request@v1.0.1
+        with:
+          destination_branch: "main"
+          pr_title: "Ready for Main"
+          pr_body: "Merge to main, commit: ${{ github.sha }}"
```

Comment:
This will fail certain checks, such as the PSI tests.
Can we actually get a meaningful PR template that also includes links to all the relevant T1 stage pages & a publicly accessible milo one for PSI? It would always be main—milo—adobecom & stage—milo—adobecom

Reply:
I would not say the PR description does not matter. Anyone can look into those PRs, the descriptions serve an important role from a git/history perspective - also the SOTs & also devs should have a few links handy to click and go through each project.

Reply:
An engineer that looks into stage->main sync, should also always be able to know what changes are about to go live and block them if any issues are encountered. We want to improve the process, not shift every responsibility away from an engineer to the SOTs

Reply:
I agree with @mokimo here: having meaningful PR descriptions will definitely be helpful for reviewers and approvers to understand the context of changes planned to be merged into `main`.

Since checks are run regardless of the branch, we should make sure they are green no matter what.